src/ex_cmds.h

```diff
@@ -1157,6 +1157,9 @@ EXCMD(CMD_perldo,	"perldo",	ex_perldo,
 EXCMD(CMD_pedit,	"pedit",	ex_pedit,
 	EX_BANG|EX_FILE1|EX_CMDARG|EX_ARGOPT|EX_TRLBAR,
 	ADDR_NONE),
+EXCMD(CMD_pbuffer,	"pbuffer",	ex_pbuffer,
+	EX_BANG|EX_RANGE|EX_BUFNAME|EX_BUFUNL|EX_COUNT|EX_EXTRA|EX_CMDARG|EX_TRLBAR,
+	ADDR_BUFFERS),
```

Comment:
Is the additional location correct here?
In alphabetical order, shouldn't it be right after "packloadall"?